services/QuillLMS/client/app/bundles/PremiumHub/shared.tsx

```diff
@@ -80,3 +81,15 @@ export function hashPayload(payloadArray: Array<any>) {
   const joinedPayload = payloadArray.join('-')
   return md5(joinedPayload)
 }
+
+export function getTimeInMinutesAndSeconds(seconds) {
+  if(!seconds) return NOT_APPLICABLE
+
+  let numminutes = Math.floor((((seconds % 31536000) % 86400) % 3600) / 60).toString();
+  let numseconds = Math.floor((((seconds % 31536000) % 86400) % 3600) % 60).toString();
```

Comment:
It would be good to have these numbers as constants so it's clear what they mean.